css/style.css

```diff
@@ -1,19 +1,223 @@
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+ 
 
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    height: 100%;  
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
 
+/* header */
 
+
+#page-nav img {
+    display: block;
+    float: left;
+    width: 30px;
+    padding: 10px 20px;
+    margin-left: 100px;
+  }
+
+#page-nav {
+    width: 100%;
+    background-color: white;
+    position: fixed; 
+}
+
+
+#page-nav label, #hamburger {
+    display: none;
+}
+  
+/* Menu Items */
+#page-nav ul {
+    
+    list-style-type: none;
+    margin: 0;
+    padding: 0; 
+    float: right;
+    
+}
+#page-nav ul li {
+    display: inline-block;
+    padding: 25px;
+    margin-right: 30px;
+}
+#page-nav ul li a {
+    color: orangered;
+    text-decoration: none;
+}
+#page-nav ul li a:hover {
+    color: black;
+    
+}
+#page-nav ul li a.active {
+    color: black;
+    
+ }
+
+/*MAIN*/
+img {
+    display: block;
+}
+
+#back {
+    display: flex;
+    justify-content: center;
+    background-image: url('/img/snow_background copy.jpg');
```

Comment:
Try and fix your image by editing the path here. 
Hint: try adding a dot or 2 at the beginning 😉